README.md

```diff
@@ -1 +1,4 @@
-# E-Commerce-Project
\ No newline at end of file
+# Dash Cloths 
+Dash cloths is a online cloths website where you can puchase  attractive cloths for men and woman 
+
+# Table of contents
```

Comment:
Add technology and directory structure. @Yshah112 @rinkishivani

Reply:
added technology and directory structure.